src/common/config.cc

```diff
@@ -1227,6 +1227,47 @@ void md_config_t::diff(
   }
 }
 
+void md_config_t::diff_setting(const md_config_t *other,
+            std::map<std::string, std::pair<std::string, std::string>> *diff, 
+            std::set<std::string> *unknown, const std::string& ceph_setting, bool show_unchanged)
```

Comment:
nit, the `std::` is not necessary, as we are `using std::string` and `using std::map` at the top of this source file.